cmd/proxy/actions/app.go

```diff
@@ -104,52 +86,44 @@ func App(conf *config.Config) (*buffalo.App, error) {
 		lggr.Infof("%s", err)
 	} else {
 		defer flushTraces()
-		app.Use(observ.Tracer(Service))
 	}
 
 	// RegisterStatsExporter will register an exporter where we will collect our stats.
 	// The error from the RegisterStatsExporter would be nil if the proper stats exporter
 	// was specified by the user.
-	flushStats, err := observ.RegisterStatsExporter(app, conf.StatsExporter, Service)
+	flushStats, err := observ.RegisterStatsExporter(r, conf.StatsExporter, Service)
 	if err != nil {
 		lggr.Infof("%s", err)
 	} else {
 		defer flushStats()
-		app.Use(observ.StatsMiddleware())
-	}
-
-	// Automatically redirect to SSL
-	app.Use(forcessl.Middleware(secure.Options{
```

Comment:
Does this break any functionality?

Reply:
@manugupt1 I'm not aware if this was used at all. I'll bring it back just in case :)